addons/manager.go

```diff
@@ -69,7 +68,7 @@ func NewAddonAgentCommand() *cobra.Command {
 		Short: "Start the addon agents",
 		Run: func(cmd *cobra.Command, args []string) {
 			ctx := ctrl.SetupSignalHandler()
-			o.RunAgent(ctx)
+			o.RunAgent(ctx, slog.New(slog.NewTextHandler(os.Stdout, nil)))
```

Comment:
We need JSON handler for production, Text handler for development. Need to use a flag or envvar to switch between them.

Reply:
Added a flag (--dev) to switch between prod and dev